src/index.js

```diff
@@ -1,13 +1,126 @@
-export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
-}
-
-// export default class BaseballGame {
-//   play(computerInputNumbers, userInputNumbers) {
-//     return "결과 값 String";
-//   }
-// }
-
-new BaseballGame();
+export default class BaseballGame { 
+
+    constructor() {
+      this.computerNum = this.randomNumGenerator();
+      this.strike = 0;
+      this.ball = 0;
+      this.result = '';
+      console.log(this.computerNum);
+    }
+  
+    randomNumGenerator() {
+      let first = Math.floor(Math.random() * 9 + 1);
+      let second = Math.floor(Math.random() * 9 + 1);
+      while (first == second) {
```

Comment:
```suggestion
      while (first === second) {
```

자바스크립트에서 == 는 값을 비교하고, ===는 타입을 기반으로 비교하기 때문에 엄격한 비교인 ===를 쓰는게 조금 더 좋아보여서 리뷰했습니다.